lib/beaker/dsl/helpers/host_helpers.rb

```diff
@@ -331,7 +331,7 @@ def upgrade_package host, package_name
         #
         # @return nil
         def add_system32_hosts_entry(host, opts = {})
```

Comment:
@joshcooper raises a good point.  

There are a few meta-questions here, including "why is there this random powershell-specific method in the hosts helpers?".  My understanding is that @anodelman is in the midst of more cleanup and consolidation of windows-related methods.

As the code exists at the moment, things inside the hosts helper are presuming that puppet is not necessarily installed.  In the bigger picture, it may well be useful to lift some methods up and out to a place where they could assume that puppet is installed.

This PR is just concerned with wrapping acceptance tests (actually, a number of these are specifically [characterization tests](http://c2.com/cgi/wiki?CharacterizationTest)) around core code that doesn't really have coverage.  After we have tests in place we should feel more empowered to think about consistency and how functionality is carried out.